lib/wallet/src/Cardano/Wallet/DB/Store/DeltaUTxO/Model.hs

```diff
@@ -0,0 +1,164 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Cardano.Wallet.DB.Store.DeltaUTxO.Model (
+    -- * Types
+    UTxOHistory,
+
+    -- * Accessors
+    getTip,
+    getFinality,
+    empty,
+    getUTxO,
+
+    -- * Changes
+    DeltaUTxOHistory (..),
+
+    -- * For testing
+    getSpent,
```

Comment:
```suggestion
```

What if we take the idea of the `Internal` module to its logical conclusion and only export this function from `….Model.Internal`? 🤔 The `….Model` module could become a thin re-export layer, which most of the logic implemented in the `….Model.Internal` module.

Reply:
I'm not sure here, but I will move it up. I was hoping the ModelSpec does not have to import from Internal.

Reply:
I decided to leave some stuff serving the store here as well. It feels high level to me